docs/src/pages/customization/palette/palette.md

```diff
@@ -193,61 +193,4 @@ Need inspiration? The Material Design team has built an [palette configuration t
 
 ## Dark mode
 
-Material-UI comes with two palette modes: light (the default) and dark.
-You can make the theme dark by setting `mode: 'dark'`.
-
-```js
-const darkTheme = createTheme({
-  palette: {
-    mode: 'dark',
-  },
-});
-```
-
-While it's only a single value change, the `createTheme` helper modifies several palette values.
-The colors modified by the palette mode are the following:
-
-{{"demo": "pages/customization/palette/DarkTheme.js", "bg": "inline", "hideToolbar": true}}
-
-### Toggling color mode
-
-You can use the React context to toggle the mode with a button inside your page.
-
-{{"demo": "pages/customization/palette/ToggleColorMode.js", "defaultCodeOpen": false}}
-
-### System preference
-
-Users might have specified a preference for a light or dark theme.
-The method by which the user expresses their preference can vary. It might be a system-wide setting exposed by the Operating System, or a setting controlled by the User Agent.
-
-You can leverage this preference dynamically with the [useMediaQuery](/components/use-media-query/) hook and the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query.
-
-For instance, you can enable the dark mode automatically:
-
-```jsx
-import * as React from 'react';
-import useMediaQuery from '@mui/material/useMediaQuery';
-import { createTheme, ThemeProvider } from '@mui/material/styles';
-import CssBaseline from '@mui/material/CssBaseline';
-
-function App() {
-  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
-
-  const theme = React.useMemo(
-    () =>
-      createTheme({
-        palette: {
-          mode: prefersDarkMode ? 'dark' : 'light',
-        },
-      }),
-    [prefersDarkMode],
-  );
-
-  return (
-    <ThemeProvider theme={theme}>
-      <CssBaseline />
-      <Routes />
-    </ThemeProvider>
-  );
-}
-```
+For more details of how you can set up a dark mode for your theme, head to the [dark mode guide](/customization/dark-mode).
```

Comment:
Otherwise, it creates a 301 redirection

```suggestion
For more details of how you can set up a dark mode for your theme, head to the [dark mode guide](/customization/dark-mode).
```

Reply:
I'm not sure about "more details", what does it refer to?